microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -489,52 +1069,98 @@ You may turn off list updates using the following group policy, [Enable componen
 
-## Thank you!  
+## Thank you  
 
-Microsoft Edge is made possible by the [Chromium][ChromiumMain] open-source project and other open-source software.  To view all of the software credits, navigate to `edge://credits`.  [Google Chrome Privacy Whitepaper][GoogleChromePrivacyWhitepaper] was used as a source for gathering related information about the Chromium open source project.  
+Microsoft Edge is made possible by the [Chromium][ChromiumMain] open-source project and other open-source software.  To display all the software credits, navigate to `edge://credits`.  [Google Chrome Privacy Whitepaper][GoogleChromePrivacyWhitepaper] was used as a source for gathering related information about the Chromium open-source project.  
 
-<!--Microsoft Edge is committed to protecting and respecting your privacy while giving you the transparency and control you deserve.  Reach out to [@MSEdgeDev][TwitterMsedgedev] on Twitter or submit feedback by opening **Settings and more (...)** > **Help and feedback** and selecting **Send feedback** with questions or comments.  -->  
+<!--  Microsoft Edge is committed to protecting and respecting your privacy while giving you the transparency and control you deserve.  Reach out to [@MSEdgeDev][TwitterMsedgedev] on Twitter or submit feedback by opening **Settings and more (...)** > **Help and feedback** and choosing **Send feedback** with questions or comments.  -->  
 
 ## Getting in touch with the Microsoft Edge team  
 
-The Microsoft Edge team is always listening to customers and values your feedback.  To provide feedback in Microsoft Edge, open **Settings and more** > **Help and feedback** and select **Send feedback**.  For Progressive Web Apps \(PWAs\), open **Settings and more (...)** and select **Send feedback to Microsoft**.  You must provide details about the feedback, but all other information is optional.  If an email is detected from your Microsoft Edge profile, it is pre-populated, along with the current URL of the site you are on and relevant diagnostic data.  The diagnostic data may include data about Microsoft Edge features you turned on and your usage of the browser.  A screenshot, files from your device, and recording of your browser may also be optionally included.  The optional screenshot, files, or recording you provide may include personally identifiable information.  The data is only used for diagnostic and product improvement purposes.  
+The Microsoft Edge team is always listening to customers and values your feedback.  To provide feedback in Microsoft Edge, complete the following actions.  
```

Comment:
"Complete the following actions" is a _perfect_ example of "robot words", as opposed to warm and relaxed.
https://docs.microsoft.com/en-us/style-guide/brand-voice-above-all-simple-human
https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice - "Read your text aloud. Does it sound like something a real person would say? Be friendly and conversational. No. Robot. Words."